requirements/requirements_nemo_text_processing.txt

```diff
@@ -1,2 +1 @@
-inflect
-regex
+nemo_text_processing==0.1.6rc0
```

Comment:
should we also remove this file and move the dependency to a better place? If all domains, including asr, nlp, tts, slu, rely on `nemo_text_processing`, then I guess `requirements.txt` could be the target place?

Reply:
BTW, I saw `requirements.txt` has already added this lib. So we are safe to remove this file, right?

Reply:
> BTW, I saw `requirements.txt` has already added this lib. So we are safe to remove this file, right?

no, `requirements.txt` doesn't have it; I'll add to `requirements_tts.txt` and remove `requirements_nemo_text_processing.txt`